dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -108,6 +108,63 @@ const decideIfAgeShouldShow = ({
 	return false;
 };
 
+const DecideFooter = ({
+	isOpinion,
+	hasSublinks,
+	displayAge,
+	renderFooter,
+}: {
+	isOpinion: boolean;
+	hasSublinks?: boolean;
+	displayAge: boolean;
+	renderFooter: Function;
+}) => {
+	if (isOpinion) {
+		if (hasSublinks) {
+			// Opinion cards with sublinks show their footers under the trailtext but
+			// without the lines, these sit by themselves at the bottom
+			return renderFooter({
+				displayAge,
+				displayLines: false,
+			});
+		} else {
+			// Opinion cards without sublinks render the whole footer including lines, outside, sitting along the very bottom of the card
+			return null;
+		}
+	} else {
```

Comment:
If I'm reading this correctly then two of the branches here are returning the same value, right? In which case presumably we could have instead:

```js
if (isOpinion && !hasSublinks) {
  return null;
}
return renderFooter({
  displayAge,
  displayLines: false,
});
```

Even though it avoids some nested conditionals, I'm slightly reluctant to suggest this because I think that the current structure gives a good way to spell out what the logic is doing in the comments. (I found the comments super helpful btw!) That said, if it's possible to keep the comments as illuminating with a simpler conditional structure, that could be nice.

Reply:
Yeah, I kind of started here but then wrote the more verbose version so that I could spell it out a bit more.

It's super hard to know what's best in these situations. My brain has been full of this for a few days now so it's difficult for me to put myself in the shoes of someone coming to this code from scratch. Happy to take a steer from others on this

Reply:
If we're not anticipating adding other conditions at the moment, then I think I might incline towards the simpler conditional. I think nested conditionals are generally harder to interpret, and it's also the sort of thing I can easily imagine someone deciding to refactor in passing, which would risk untethering the logic from the comments. So I don't have strong feelings on this but fwiw I'd be inclined to go with the simpler logic (although also definitely keeping the helpful comments that explain the rationale!)